package/base-files/files/lib/upgrade/common.sh

```diff
@@ -71,6 +71,7 @@ _vn() {
 
 v() {
 	_v "$(date) upgrade: $@"
+	logger -p info -t sysupgrade "$@"
```

Comment:
Looks broken to me if it only works for v(), but not for vn() ...

Reply:
This vn pattern is currently used in the following places in openwrt base

[stage2](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/upgrade/stage2):
https://github.com/openwrt/openwrt/blob/d33cd383ed64f3b387c8e3111c3c0c663763c0ea/package/base-files/files/lib/upgrade/stage2#L77-L102

For this I don't see any problem with your suggestion to write that also to the syslog.

[fwtool](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/upgrade/fwtool.sh):
https://github.com/openwrt/openwrt/blob/d33cd383ed64f3b387c8e3111c3c0c663763c0ea/package/base-files/files/lib/upgrade/fwtool.sh#L82-L88

There I have more of a problem dealing with it properly.
Because if I do it this way, it only says "Supported devices:" in the log, which is not correct in my opinion.

Reply:
@adschm I have now adjusted the vn use in stage2 in the function kill_remaining, so that this now also uses the function v. The output for the supported device does not make any sense from my point of view in the syslog output. However, it already says that the device is not supported because this used the v function.